src/content_cao.cpp

```diff
@@ -1786,6 +1831,14 @@ void GenericCAO::processMessage(const std::string &data)
 	}
 }
 
+void GenericCAO::knockback(const v3f &direction, f32 time_knockback)
+{
+	m_knockback_timer = time_knockback;
```

Comment:
Sorry, i misunderstood as explained below.

So i guess the player receiving knockback has their controls disabled because they have to be placed at the final position of the clientside knockback animation, so cannot be allowed to move during flight to a different position?